src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -58,4 +72,23 @@ public void startExecuting()
 
         citizen.getNavigator().tryMoveToXYZ((double) pos.getX() + 0.5D, (double) pos.getY(), (double) pos.getZ() + 0.5D, 1.0D);
     }
+
+    /**
+     * While going home play a goHome sound for the specific worker by chance.
+     */
+    private void playGoHomeSounds()
+    {
+        final Random rand = new Random();
+
+        final int chance = rand.nextInt(ONE_IN_A_THOUSAND);
+
+        if(chance <= 1)
+        {
+            if (citizen.getWorkBuilding() != null && ("fisherman").equals(citizen.getWorkBuilding().getJobName()))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1066)